@here/harp-materials/lib/DisplacementFeature.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HiddenThreeJSMaterialProperties } from "./MapMeshMaterials";
+
+/**
+ * Parameters used when constructing a new implementor of [[DisplacementFeature]].
+ */
+export interface DisplacementFeatureParameters {
+    /**
+     * Texture used for vertex displacement along their normals.
+     */
+    displacementMap?: THREE.Texture;
+}
+
+/**
+ * Interface to be implemented by materials that use displacement maps to overlay geometry
+ * on elevation data.
+ */
+export interface DisplacementFeature extends HiddenThreeJSMaterialProperties {
+    displacementMap: THREE.Texture | null;
```

Comment:
I've changed the displacementMap type to match the one in THREE.js MeshStandardShader (THREE.Texture | null). That way the interface is homogeneous across all materials with the displacement feature and they can be handled in the same way (using displacementMap getter and setter).

Reply:
I am wondering if all c-tors are handling DisplacementFeatureParameters correctly now, cause we have mismatch here, undefined !== null.
Please verify it.

Reply:
I double checked, everything's fine.